src/common/util/CMakeLists.txt

```
@@ -33,6 +33,9 @@ add_library(${TARGET_NAME} STATIC ${LIBRARY_SRC} ${PUBLIC_HEADERS})
 add_library(openvino::util ALIAS ${TARGET_NAME})
 
 target_link_libraries(${TARGET_NAME} PRIVATE ${CMAKE_DL_LIBS})
+if (WIN32)
+    target_link_libraries(${TARGET_NAME} PRIVATE Shlwapi.lib)
```

What if:
```suggestion
    target_link_libraries(${TARGET_NAME} PRIVATE Shlwapi)
```

Do we have a dll Shlwapi library too?

---

why do we need it at all?

---

Yes, dll also available, will be changed